apps/balancer-tools/src/app/apr/(utils)/calculateRoundAPR.ts

```diff
@@ -1,10 +1,38 @@
 import * as balEmissions from "#/lib/balancer/emissions";
+import { Pool } from "#/lib/balancer/gauges";
 
+import BalancerAPI from "./balancerAPI";
+import { getBALPriceByRound } from "./getBALPriceByRound";
+import { getPoolRelativeWeight } from "./getRelativeWeight";
 import { Round } from "./rounds";
 
 const WEEKS_IN_YEAR = 52;
 
-export default function calculateRoundAPR(
+export async function calculateAPRForPool({
```

Comment:
let's be consistent with our naming convention. see the definition below in`calculateRoundAPR`
```suggestion
- export async function calculateAPRForPool({
+ export async function calculatePoolAPR({
```

Reply:
renamed to calculatePoolStats